### PR TITLE
feat(model): custom gravity gear extension model

### PR DESCRIPTION
**Pull request by tyler58546**
## Summary of Changes

- Added custom animated gravity gear extension model
- Adjusted phone cord

## Screenshots (if necessary)

image
image
image

## Additional context

Discord username (if different from GitHub):

## Testing instructions

- Make sure there are no issues with the model/animation
- Make sure the gravity gear extension still works

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: src/model/models.json [hunk 1]
```diff
@@ -90,6 +90,16 @@
       {
         "gltf": "./glass/glass.gltf",
         "bin": "./glass/glass.bin"
+      },
+      {
+        "gltf": "./cockpit/cockpit.gltf",
+        "bin": "./cockpit/cockpit.bin",
+        "maxLod": 0
+      },
+      {
```
Comment (Saschl) [inline src/model/models.json:99]: Those are missing in the PR ;)

File: src/model/models.json [hunk 2]
```diff
@@ -90,6 +90,16 @@
...
+        "gltf": "./cockpit/gearemer.gltf",
+        "bin": "./cockpit/gearemer.bin",
```
Comment (Saschl) [inline src/model/models.json:101]: I think "gearemer.gltf" and "gearemer.bin" are missing in the PR ;)

Comment (tyler58546) [reply src/model/models.json:101]: oops

**Review by 2hwk — APPROVED** [review]

**Comment by Mack674** [comment]
Date             : 29/07/2022

Testing Process: 
- Spawned C&D at LFMN aiport
- Loaded and departed
- During approach used emergency extension handle
- Checked animation
- Landed

Negatives: N/A      

Testing Results: Passed

Conclusions: The animation works fine in my opinion. The extension from the outside also looks good. Low video lag and low FPS are my computer's fault 😉 

Media:

https://user-images.githubusercontent.com/62873705/181800165-c2dccc96-0bb0-4f42-b695-bc4c8a4e81d2.mp4

**Pull request merged**